styles/full.module.css

```diff
@@ -8,19 +8,29 @@
   padding: 0px;
   margin: 0px;
 }
-
 .sectionContent {
   margin: 0 auto;
   padding: 0 120px;
 }
-
 .gridSection {
   display: grid;
   grid-template-columns: repeat(12, 1fr);
   column-gap: 24px;
   max-width: 1240px;
   margin: 0 auto;
 }
+.flexSection {
+  display: flex;
+  max-width: 1240px;
+  margin: 0 auto;
+}
+.dark {
+  color: #ffffff;
+  background: linear-gradient(68.66deg, #0f1011 0%, #010242 100%);
```

Comment:
Fallback background color/mid value would be good for cases where linear-gradient isn't supported:

```
background: #0f1011; // or whatever mid/fallback color you want
background: linear-gradient(68.66deg, #0f1011 0%, #010242 100%);
```

Reply:
Ya, I did check the support for [linear-gradient](https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient()) and it's pretty solid, but this is easy to add in for safety just in case so :+1: